src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
   `server.address` when it has default values (`80` for `HTTP` and
   `443` for `HTTPS` protocol).
   ([#5419](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5419))
+* Fixed an issue for spans when `http.request.method_original` attribute was not
```

Comment:
Could you please explain the issue in detail and why it should be fixed?

Reply:
It is linked in the PR description: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1607#discussion_r1534215839

I prefer to avoid duplication of SemConv in the changelog. If you think it otherwise, could you please prepare better wording to include?

This fix should have low footprint in my opinion. Usually HTTP methods are send in canonical form.

Reply:
Fixed an issue where the `http.request.method_original` attribute was not set on activity. 
Now, when `http.request.method` is set and the original method is converted to its canonical form (e.g., Get is converted to GET), the original value Get will be stored in `http.request.method_original`.

Reply:
bb346b197b88f0fc2d83a09eb126f075b3f74361